### ISSUE TITLE
If the ~/.elm folder has been cleared the tool fails.

### ISSUE THREAD

**Issue by supermario** [wolfadex/elm-license-finder#2]
```
$ elm-license-finder
internal/fs/utils.js:227
    throw err;
    ^

Error: ENOENT: no such file or directory, open '/Users/mario/.elm/0.19.1/packages/BrianHicks/elm-trend/2.1.2/elm.json'
    at Object.openSync (fs.js:446:3)
    at Object.readFileSync (fs.js:348:35)
    at buildDependencyTree (/usr/local/lib/node_modules/elm-license-finder/index.js:106:23)
    at Object.<anonymous> (/usr/local/lib/node_modules/elm-license-finder/elm-license-finder.js:6:22)
    at Module._compile (internal/modules/cjs/loader.js:1128:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
    at Module.load (internal/modules/cjs/loader.js:983:32)
    at Function.Module._load (internal/modules/cjs/loader.js:891:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/Users/mario/.elm/0.19.1/packages/BrianHicks/elm-trend/2.1.2/elm.json'
}
```

It might be nicer to catch failures and let the user know the project needs to be compiled before `elm-license-finder` is run.

**Comment by supermario** [comment]
Further to this, it still fails after install on `0.19.0` due to the hardcoded Elm version paths:

https://github.com/wolfadex/elm-license-finder/blob/master/index.js#L81-L92

Perhaps these should use the `"elm-version"` field from elm.json.

`ELM_HOME=~/.elm/0.19.0/package elm-license-finder` works as a workaround for now.

**Comment by wolfadex** [comment]
These will be resolved with v1.1.0 which I'll publish when I get home tonight.

I also added a few other helpful error messages for known/easily solvable errors.

Thank you @supermario for filing the bug

**Comment by wolfadex** [comment]
Resolved in 1.1.0
